airqo-platform/AirQo-api
# Hourly pm averages

- What does this PR do?
This PR consumes an endpoint developed by @sserurich which generates hourly averages of pollution values for each channel.
- What JIRA task is related to this PR?
The link to the JIRA board task is [here](https://airqoteam.atlassian.net/browse/DAT-73?atlOrigin=eyJpIjoiMTRhMWRmMDczYzI3NGEzYjhjOTI1YjcwZjJkMDlkYTEiLCJwIjoiaiJ9).
- How do I test out this PR?
Git clone this repo `cd AirQo-api/src/data-mgt` Then run `npm install` to get all the necessary dependencies Afterwards run `npm run test` to test out the end points using any REST client tool available to you.
- Which end points should are ready for testing?
partial URI: /api/v1/feeds/hourly/<CHANNEL ID>
base URI: https://data-manager-dot-airqo-250220.appspot.com
More details about the endpoint can be found in the [API documentation](https://docs.google.com/spreadsheets/d/1-NGTDBtTcIi8Fg2UYSWjcK6geKrZdopz5kxRFsCEMPs/edit#gid=757070984).
